package.json

```diff
@@ -38,8 +38,6 @@
     "jest": "^29.7.0",
     "ts-jest": "^29.1.1",
     "typescript": "^5.0.2",
-    "vite": "^4.3.2",
-    "vite-plugin-pwa": "^0.15.1",
```

Comment:
@john-wayne630 Make sure the vite-plugin-pwa package is not used. Return it if it's in use.